### ISSUE TITLE
Priority list for a showcase

### ISSUE THREAD

**Issue by konnov** [informalsystems/quint#541]
This pinned issue contains the issues that we should prioritize from the product perspective, in order to produce a good showcase example. These issues are ordered by their importance. However, if some of lower priority issues could be solved much faster than those of higher priority, we may change the order.

The current showcase example is the Cosmos bank module, leaving in a branch, since it contains plenty of TODOs: https://github.com/informalsystems/quint/blob/igor/cosmos-bank/examples/cosmos-bank/bank.qnt

Note that this list does not take into account any work on: design, refactoring, technical debt, etc. Thus, we should be careful about not chasing only this list, but also having a look at technical priorities.
 
 - [x] Tutorials:
   - [x] #510
   - [x] #438
   - [x] #495
   - [x] #522
   - [x] #682 
 - [x] Implement/fix support for multiple modules:
   - [x] #523
   - [x] #532
   - [x] #531
   - [x] #496
   - [x] Design and implement imports from other files: #241
 - [x] Implement support for constants in module instances (depends on #523?): #528
 - [x] Notebook-style editing, Quint <-> markdown
    - Using [lmt](https://github.com/driusan/lmt)
 - [x] Improve the runs operators for tests:
   - [x] #542
   - [x] #543
 - ~[ ] Redesign algebraic data types and implement them: #539~

**Comment by konnov** [comment]
Closing this issue. All features for this showcase example have been implemented, except ADTs that require more thought and work, but there are simple workarounds.
